### ISSUE TITLE
Interpolation of multiline expressions

### ISSUE THREAD

**Issue by mrkkrp** [yesodweb/yesod#1189]
Is there a way to split expression to interpolate into multiple lines, like this:

```
^{myWidgetThatTakesFiveThings (Just oneThing) (Just anotherThing) (Just oneMoreThing)
    thisOneOnANewLine
    andMaybeOneMoreLine}
```

I believe it might help readability is some cases. (Escaping with `\` does not seem to work.)

**Comment by snoyberg** [comment]
I agree that this could be an improvement to the current syntax, but to my
knowledge it is not supported without changes to shakespeare.
